docker-compose.yml

```diff
@@ -36,6 +36,15 @@ services:
       - DEMO_TOKEN=demo-random-string
       - DEMO_PERSON=Demo
       - DEMO_EMAIL=demo@demo.local
+    healthcheck:
```

Comment:
Perhaps this check can be moved to web-init service, because "web" service depends on it

Reply:
Better leave it here, the init container will go down anyway when its done, also celery depends on the "web" service health check: 
```
    depends_on:
      web:
        condition: service_healthy
```